deploy/rp-production-global.json

```diff
@@ -27,6 +27,16 @@
             "condition": "[parameters('fullDeploy')]",
             "apiVersion": "2019-05-01"
         },
+        {
+            "sku": {
+                "name": "Standard_LRS"
+            },
+            "location": "[resourceGroup().location]",
+            "name": "arorpversion",
```

Comment:
storage account names are globally unique.  Need to parameterise this so that it works in INT and PROD - the actual value should come from the RP config repo.